websockets/feeds/feed.go

```diff
@@ -65,6 +67,8 @@ type Feed struct {
 	moderatePost chan moderationMessage
 	// Let sent sync counter
 	lastSyncCount syncCount
+	// trigger sending IP count/cinema status, fired from cinema feed loop
+	sendIPCountChan chan int
```

Comment:
A `Feed` and and a `cinemaFeed` don't have linked lifetimes, thus it is possible for this channel to cause blocked and thus leaked goroutines. Please use the global `SendTo` function instead and delete this channel.

Reply:
I don't understand what is your solution here. `SendTo` buffers a message to all clients, but in order to make it I need information from main feed (IP, IP active count), which can only be obtained from its goroutine.

Maybe guard those two values with mutex in a separate struct as I've done in cinema feed and make `sendIPCount` thread safe. Another option is to send the same message from cinema feed with values from main feed set to null, then client won't update them or entirely move cinema status to a different websocket message.

Reply:
You're right - `SendTo()` is not what is needed. A better solution would be adding a reasonable buffer to `sendIPCountChan`, say 16, and then using a non-blocking select send in the cinema feed. In case it's unclear, I mean this:

```go
select {
case f.sendIPCountChan <- count:
default: 
// Log error
}
```

Reply:
Ok, sounds good, if I understand correctly that additional select is to catch error when the buffer is full?